lib/Logging.js

```diff
@@ -39,24 +39,22 @@ class Logging {
             }));
         }
 
-        if(config.files === undefined) {
-            return;
-        }
-
-        let i = 0;
-        for (let file of config.files) {
-            const filename = Object.keys(file)[0];
-            const level = file[filename];
-            i++;
-            this.transports.push(new (winston.transports.DailyRotateFile)({
-                filename,
-                datePattern: "YYYY-MM-DD",
-                name: `logfile` + i,
-                formatter: formatterFn,
-                level,
-                timestamp: timestampFn,
-                maxFiles: config.maxFiles > 0 ? config.maxFiles : undefined
-            }));
+        if(config.files !== undefined) {
```

Comment:
Is this actually a functional change? It doesn't seem like it.

Reply:
It's because of the old code:
```
if(config.files === undefined) {
            return;
}
```
which would cause  ``wrapper.setLogger`` to not be called if ``config.files`` wasn't defined. So a console only logging config would just not work.

Reply:
Oh, I missed line 59 onwards, ignore me.